vue/components/ui/organisms/input-list/input-list.vue

```diff
@@ -155,7 +159,7 @@ export const InputList = {
     computed: {
         buttonContainerStyle() {
             const base = {};
-            if (this.buttonAddRow === "top") {
+            if (this.buttonAddRow === "top" && this.header) {

```

Comment:
Use a new prop for this so that the position of the add button is not defined in terms of the header's visibility. Something like `addPosition` (top/bottom)

Reply:
But that is what the `buttonAddRow` prop already does. We just added the behaviour to automatically place it on the bottom if there's no header, otherwise there was an overlap between the button to add a new list, and the button to remove the first element of the list. 

Without that check, this is what happens when the `buttonAddRow` is set to `top`, and the `header` is de-selected:
![image](https://user-images.githubusercontent.com/22790704/118115045-5250b800-b3e0-11eb-9e5d-1a69153cd279.png)

Reply:
This has changed in the latest commits, now if the `header` prop is set to false and the `buttonAddRow` is set to `top`, the labels are removed, but the button is still in the same position. The header is only fully removed if the `header` prop is false, and the `buttonAddRow` is set to `bottom`.